18-minimal/Dockerfile.c8s

```diff
@@ -45,12 +45,11 @@ LABEL summary="$SUMMARY" \
       com.redhat.component="${NAME}-${NODEJS_VERSION}-minimal-container" \
       name="sclorg/$NAME-$NODEJS_VERSION-minimal-c8s" \
       version="1" \
-      com.redhat.license_terms="https://www.redhat.com/en/about/red-hat-end-user-license-agreements#UBI" \
```

Comment:
Nodejs 18-minimal is UBI based, right? license should be present here.

Reply:
This one is CentOS Stream based, so I don't think we need a license tag.